sample/common/src/commonMain/kotlin/com/trikot/metaviews/sample/viewmodels/home/ButtonsViewModel.kt

```diff
@@ -35,6 +35,18 @@ class ButtonsViewModel(navigationDelegate: NavigationDelegate): ListViewModel {
                     RichTextRange(IntRange(34, 45), StyleTransform(StyleTransform.Style.BOLD_ITALIC))
                 )).just()
         },
+        MutableHeaderListItem(".richText with color"),
+        MutableMetaButtonListItem().also {
+            it.button.textColor = MetaSelector(Color(123, 123, 123), Color(0   , 0, 0)).just()
+            it.button.richText = RichText("normal, bold, underlined, italic, bold italic",
+                listOf(
+                    RichTextRange(IntRange(0, 8), StyleTransform(StyleTransform.Style.NORMAL)),
```

Comment:
The `IntRange` includes the endPositionIndex which makes the 8th char normal and bold in this case. Also, Kotlin's `..` operator could have been used to create the `IntRanges` here.